extensions/applicationinsights-analytics-js/src/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -647,7 +647,7 @@ export class ApplicationInsights extends BaseTelemetryPlugin implements IAppInsi
                         _window.addEventListener(_self.config.namePrefix + "locationchange", () => {
                             if (_properties && _properties.context && _properties.context.telemetryTrace) {
                                 _properties.context.telemetryTrace.traceID = generateW3CId();
-                                _properties.context.telemetryTrace.name = _location && _location.pathname || "_unknown_";
+                                _properties.context.telemetryTrace.name = (_location && _location.pathname).concat(_location.hash) || "_unknown_";
```

Comment:
location does not exist in IE8 and below or Firefox < v14 etc.
To avoid a crash we need to ensure that location exists (which is happening for pathname) 
-- additional, if pathname is not defined this will throw because concat is not a function of undefined.

Reply:
Although, because this is in the location change event that "unlikely"

Reply:
Do we have an Issue for this?
If so can we please include the number in the commit. And this *may* also not be something everyone wants as a change. So we should add this as a config, so it can be disabled for users that don't want this capability

Reply:
@MSNev thanks for the review. Yes there's an open issue here: #1393 
Should I added it in the comment above, do you usually add issue number in each commit? 

I updated the check, can you please have a second look - as for the config check - the issue says this should be added when `enableAutoRouteTracking` is true, I assume there's no need to add another config? But I do get your point. Should we add new config for it ? Thanks!